src/Remapping/distributed_remapping.jl

```diff
@@ -192,10 +191,19 @@ end
 
 
 """
-   interpolate(remapper, field)
+   interpolate(remapper, field; interpolate_topography = false)
 
 Interpolate the given `field` as prescribed by `remapper`.
 
+
+Keyword arguments
+==================
+
+- `interpolate_topography`: When `true`, the `target_zcoords` to `remapper` are interpreted
+                            as "from mean sea level" (instead of "from surface") and
+                            topography is interpolated. `NaN`s are returned for values that
+                            are below the surface.
+
```

Comment:
A couple of points:
- elsewhere in ClimaCore we use "hypsography" not "topography"
- the distinction isn't actually MSL vs surface, but whether the Z coordinate corresponds to the "reference Z" (i.e. constant level) or the "physical Z" (constant altitude). 

So how about
```suggestion
- `physical_z`: When `true`, interpolate to the physical z coordinates, taking into account hypsography.
      If `false` (the default), interpolation will be based on the reference z coordinate, i.e. will correspond to constant model levels.
      `NaN`s are returned for values that are below the surface.

```